src/containers/org/OrgObjectPermissionsContainer.js

```diff
@@ -0,0 +1,151 @@
+/*
+ * @flow
+ */
+
+import React, { useEffect, useMemo, useState } from 'react';
+
+import styled from 'styled-components';
+import { List } from 'immutable';
+import { Types } from 'lattice';
+import {
+  AppContentWrapper,
+  CheckboxSelect,
+  SearchInput,
+  Typography,
+} from 'lattice-ui-kit';
+import { useRequestState } from 'lattice-utils';
+import { useDispatch, useSelector } from 'react-redux';
+import { RequestStates } from 'redux-reqseq';
+import type {
+  Ace,
+  Organization,
+  PermissionType,
+  UUID,
+} from 'lattice';
+import type { RequestState } from 'redux-reqseq';
+
+import {
+  ActionsGrid,
+  CrumbItem,
+  CrumbLink,
+  Crumbs,
+  Divider,
+  PlusButton,
+  Spinner,
+  StackGrid,
+} from '../../components';
+import { GET_ORGANIZATION_OBJECT_PERMISSIONS, getOrganizationObjectPermissions } from '../../core/permissions/actions';
+import { resetRequestState } from '../../core/redux/actions';
+import { PERMISSIONS } from '../../core/redux/constants';
+import { selectObjectPermissions, selectOrganization } from '../../core/redux/selectors';
+import { Routes } from '../../core/router';
+import { ObjectPermissionsCardStack } from '../permissions';
+import type { ReactSelectOption } from '../../types';
+
+const { PermissionTypes } = Types;
+
+const PERMISSION_TYPE_OPTIONS = [
+  { label: PermissionTypes.OWNER.toLowerCase(), value: PermissionTypes.OWNER },
+  { label: PermissionTypes.READ.toLowerCase(), value: PermissionTypes.READ },
+  { label: PermissionTypes.WRITE.toLowerCase(), value: PermissionTypes.WRITE },
+  { label: PermissionTypes.LINK.toLowerCase(), value: PermissionTypes.LINK },
+  { label: PermissionTypes.MATERIALIZE.toLowerCase(), value: PermissionTypes.MATERIALIZE },
+];
+
+const SearchFilterAssignPermissionsGrid = styled(ActionsGrid)`
+  grid-template-columns: 2fr minmax(200px, 1fr) auto;
+`;
+
+const OrgObjectPermissionsContainer = ({
+  organizationId,
+} :{
+  organizationId :UUID;
+}) => {
+
+  const dispatch = useDispatch();
+  const [filterByPermissionTypes, setFilterByPermissionTypes] = useState([]);
+  const [filterByQuery, setFilterByQuery] = useState('');
+
+  const getOrgObjectPermissionsRS :?RequestState = useRequestState([PERMISSIONS, GET_ORGANIZATION_OBJECT_PERMISSIONS]);
+
+  const organization :?Organization = useSelector(selectOrganization(organizationId));
+
+  const key = useMemo(() => List([organizationId]), [organizationId]);
```

Comment:
wondering if this optimization is necessary since this doesn't seem to be an expensive computation

Reply:
I wonder this as well... yea, I'll get rid of it since it's unlikely to be much of an optimization. Plus, the docs say:

> You may rely on useMemo as a performance optimization, not as a semantic guarantee. In the future, React may choose to “forget” some previously memoized values and recalculate them on next render, e.g. to free memory for offscreen components. Write your code so that it still works without useMemo — and then add it to optimize performance.